examples/unet_segmentation_3d.py

```diff
@@ -104,17 +105,40 @@ def _loss_fn(i, j):
 
 device = torch.device("cuda:0")
 
-trainer = create_supervised_trainer(net, opt, _loss_fn, device, False)
+trainer = create_supervised_trainer(net, opt, _loss_fn, device, False,
+                                    output_transform=lambda x, y, y_pred, loss: [y_pred, loss.item()])
 
 checkpoint_handler = ModelCheckpoint('./', 'net', n_saved=10, save_interval=3, require_empty=False)
 trainer.add_event_handler(event_name=Events.EPOCH_COMPLETED, handler=checkpoint_handler, to_save={'net': net})
 
 
 @trainer.on(Events.EPOCH_COMPLETED)
 def log_training_loss(engine):
-    print("Epoch", engine.state.epoch, "Loss:", engine.state.output)
+    writer.add_scalar('Loss/train', engine.state.output[1], engine.state.epoch)
+
+    ones = torch.ones(engine.state.batch[1][0].shape, dtype=torch.int32)
+    first_image_tensor = engine.state.output[0][1][0].detach().cpu()
```

Comment:
Seems it's **output** data instead of **image**, I think "image" is confusing for input.
Can we change it to "first/second/third_output_tensor"?
Thanks.